dashboard/src/components/Config/AppRepoList/AppRepoAddDockerCreds.v2.tsx

```diff
@@ -182,7 +182,12 @@ export function AppRepoAddDockerCreds({
             </div>
           </div>
           <div className="clr-form-separator">
-            <CdsButton type="button" disabled={creating} onClick={handleInstallClick}>
+            <CdsButton
+              type="button"
+              disabled={creating}
+              onClick={handleInstallClick}
+              id="submit-docker-creds"
```

Comment:
here I need to use a selector for the ID, not sure why `await expect(page).toClick("cds-button", { text: "Submit" });`  is unable to find the button...

Reply:
I think it does find *a* button, just not the one we want it to find:

![cds-button](https://user-images.githubusercontent.com/497518/93539155-4452b080-f993-11ea-9893-1fc6aebee3f4.png)

See below, but the `cds-button` I'm looking at here isn't the actual one we want to click on (but is probably the one which CI clicked on as it's the first in the page). Guesses?

Reply:
ah, got it, it can be that. The thing is that the error is misleading and it was saying `Node is either not visible or not an HTMLElement` but I guess that is because whatever the selector returns, is returning an array rather than a single node